tests/fixtures/elevation_geonames_conn.yml

```diff
@@ -1,24 +1,24 @@
 http_interactions:
 - request:
     method: get
-    uri: http://api.geonames.org/srtm3?lats=50.01%2C51.01&lngs=10.2%2C11.2&username=<geonames_user>
+    uri: http://api.geonames.org/srtm3?lats=50.01%2C51.01&lngs=10.2%2C11.2&username=<gbif_user>
```

Comment:
No not intended. My `geonames_user` and `gbif_user` names just happen to be the same so it works in testing.

Reply:
I am actually not sure why this is happening. I didn't change any of the `elevation.r` code or the tests, so I think it might be something related to `vcr`. The tests and function both use `geonames_user` as default. 
https://github.com/ropensci/rgbif/blob/5a1c48ca606ef616819b5da0d3e0ae96880d08f1/R/elevation.r#L172